CMSIS/DoxyGen/Pack/src/devices_schema.txt

```diff
@@ -1587,21 +1587,25 @@ This element can be defined on various levels. Inner memory elements supersede o
     <td>optional</td>
   </tr>
   <tr>
-    <td>id (deprecated in Version 1.4.0)</td>
-    <td>(deprecated Version 1.4.0) Identifier of the memory region consisting of a type indicator and an index (for example, \token{IRAM1}).
-    Predefind values can be selected as defined in \ref MemoryIDTypeEnum.</td>
-    <td>MemoryIDTypeEnum</td>
+    <td>id (deprecated since version 1.4.0)</td>
+    <td>Identifier of the memory region consisting of a type indicator and an index (for example, \token{IRAM1}).</td>
+    Predefined values can be selected as defined in \ref MemoryIDTypeEnum.</td>
+    <td>\ref MemoryIDTypeEnum</td>
     <td>optional</td>
   </tr>
   <tr>
-    <td>name (new in Version 1.4.0)</td>
-    <td>unique name of the memory (new in Version 1.4.0) to be used in conjunction with <em>access</em>
+    <td>name (new since version 1.4.0)</td>
+    <td>
+      <p>Unique name of the memory to be used in conjunction with <em>access</em>.</p>
+      <p>If a memory with the same name is already defined in a parent scope, the parent one is extended/overwritten.</p>
+      <p>Backward compatibility: If no name attribute is given but id is still present, the given id is used as the name.</p>
```

Comment:
Please put id (twice) between <em></em> to make it clear that we are talking about the attribute "id".